is/custom_jsonable_test.ts

```diff
@@ -0,0 +1,113 @@
+import { assertEquals } from "@std/assert";
+import { isCustomJsonable } from "./custom_jsonable.ts";
+
+export function buildTestcases(): readonly [name: string, value: unknown][] {
+  return [
+    ["undefined", undefined],
+    ["null", null],
+    ["string", ""],
+    ["number", 0],
+    ["boolean", true],
+    ["array", []],
+    ["object", {}],
+    ["bigint", 0n],
+    ["function", () => {}],
+    ["symbol", Symbol()],
+  ] as const;
+}
```

Comment:
If `name` is of type `string`, we may make a mistake in writing `case "..."`.
How about doing it like this:
```suggestion
export function buildTestcases() {
  const testcases = [
    ["undefined", undefined],
    ["null", null],
    ["string", ""],
    ["number", 0],
    ["boolean", true],
    ["array", []],
    ["object", {}],
    ["bigint", 0n],
    ["function", () => {}],
    ["symbol", Symbol()],
  ] as const satisfies [name: string, value: unknown][];
  type Name = typeof testcases[number][0];
  return testcases as readonly [name: Name, value: unknown][];
}
```

Reply:
```ts
export function buildTestcases() {
  return [
    ["undefined", undefined],
    ["null", null],
    ["string", ""],
    ["number", 0],
    ["boolean", true],
    ["array", []],
    ["object", {}],
    ["bigint", 0n],
    ["function", () => {}],
    ["symbol", Symbol()],
  ] as const satisfies readonly (readonly [name: string, value: unknown])[];
}
```

is enough and I'll use it.